backend/fms_core/filters.py

```diff
@@ -175,10 +176,17 @@ class Meta:
 
 class DatasetFilter(GenericFilter):
     release_flag = django_filters.NumberFilter(method="release_flag_filter")
+    latest_release_update = django_filters.CharFilter(method="latest_release_update_filter")
 
     def release_flag_filter(self, queryset, name, value):
         return queryset.filter(release_flag=value)
 
+    def latest_release_update_filter(self,queryset, name, value):
```

Comment:
your filter is not generic enough... Maybe make a filter released_within_hours and capture the value passed to the filter so you can filter from X hours to the current time instead of a fixed time ?